### ISSUE TITLE
only change prefab.m_canCollide for the active prefab

### ISSUE THREAD

**Issue by boformer** [earalov/Skylines-NoPillars#2]
Changing the m_canCollide setting for all prefabs causes problems with the zoning grid and trees.

E.g. when I place an elevated rail segment, all road segments near it will be updated: Trees on the road will reappear and the zoning grid will appear on the roads.

You can use the code from NetworkSkins to detect the active prefab of the nettool

**Comment by bloodypenguin** [comment]
I know how to detect the active prefab. But the problem is that the problem wont' be gone if I'll be able to detect the active prefab because if near segments will be of the same type of prefab they will be updated anyway. And most crossroads have same type if roads crossing.

**Comment by boformer** [comment]
That's right, but it would still be useful for train tracks stacked on roads.
